test/tools/unified-spec-runner/operations.ts

```diff
@@ -793,9 +817,25 @@ operations.set('runCursorCommand', async ({ entities, operation }: OperationFunc
 operations.set('createCommandCursor', async ({ entities, operation }: OperationFunctionParams) => {
   const collection = entities.getEntity('db', operation.object);
   const { command, ...opts } = operation.arguments!;
+  switch (opts.cursorType) {
+    case 'tailableAwait':
+      opts.tailable = true;
+      opts.awaitData = true;
+      break;
+    case 'tailable':
+      opts.tailable = true;
+      break;
+    default:
+      break;
+  }
+  delete opts.cursorType;
```

Comment:
```suggestion
  const { command, cursorType, ...opts } = operation.arguments!;
  switch (cursorType) {
    case 'tailableAwait':
      opts.tailable = true;
      opts.awaitData = true;
      break;
    case 'tailable':
      opts.tailable = true;
      break;
    default:
      break;
  }
```

If you destructure the cursor type too, there shouldn't be a reason to delete it from `opts`.